server/server.ts

```diff
@@ -95,3 +91,35 @@ function openBrowser(port) {
 
   childProcess.exec(`${start} ${url}`);
 }
+
+function isAngularProject(res) {
```

Comment:
this function doesn't even have to get any parameter now. we won't use `res` at all

Reply:
return `bool`

Reply:
change the name to `angularProjectChecker`